### PR TITLE
Dx 1295 llm cost comparison

### PR DESCRIPTION
**Pull request by joschan21**

### CODE REVIEW

**Comment by CahidArda** [comment]
I will check if the ratelimit works correctly before merging

**Comment by fahreddinozcan** [comment]
Maybe we can give a reference to Workflow pricing, since it's not that obvious. How did we calculate that money?

https://upstash.com/docs/qstash/workflow/pricing

File: examples/workflow/image-gen-with-workflow/app/page.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+export default function Page() {
+  const [start, setStart] = useState<boolean>(false)
+  const [showCode, setShowCode] = useState<boolean>(false)
+  const [promptIndex, setPromptIndex] = useState<number>(1)
+
+  useEffect(() => {
+    setPromptIndex(Math.floor(Math.random() * PROMPTS.length))
+  }, [])
+
+  const onGitHubClick = () => {}
```
Comment (fahreddinozcan) [inline examples/workflow/image-gen-with-workflow/app/page.tsx:21]: we may remove this or update

File: examples/workflow/image-gen-with-workflow/app/page.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+            className={cx(start && 'opacity-30')}
+          >
+            <IconPlayerPlayFilled size={20} />
+            Start Comparison
+          </Button>
+          <Button
+            variant="secondary"
+            onClick={() => {
+              window.open(
+                'https://github.com/upstash/qstash-js/tree/DX-1295-llm-cost-comparison/examples/workflow/image-gen-with-workflow',
```
Comment (fahreddinozcan) [inline examples/workflow/image-gen-with-workflow/app/page.tsx:40]: we should update this with main branch

**Review by fahreddinozcan — APPROVED** [review]

**Pull request merged**